modules/spring-boot-autoconfigure/examples/main/resources/application.yml

```diff
@@ -0,0 +1,24 @@
+ignite:
```

Comment:
It's fine, but I would use another path like Kafka, elastic and etc. 

Cloud:
```
spring.cloud.config:
  uri: ...
  enabled: ...
  failFast: ...
```
Kafka:
```
spring:
  metrics:
    prometheus:
      enabled: true
  kafka:
    consumer:
      bootstrap-servers:          ***
      group-id:                   ***
      client-id:                  ***
      auto-offset-reset:          earliest
      max-poll-records:           100
      enable-auto-commit:         true
    producer:
      key-serializer: ***
      value-serializer: ***
    listener:
      concurrency:       
 ```

What do you think?

Reply:
Why do we need some additional properties for Ignite configuration?